src/lightkurve/io/eleanorlite.py

```diff
@@ -0,0 +1,74 @@
+"""Reader for GSFC-ELEANOR-LITE light curve files. 
+Details can be found at https://archive.stsci.edu/hlsp/eleanor
```

Comment:
The URL of GSFC-ELEANOR-LITE is actually:
https://archive.stsci.edu/hlsp/gsfc-eleanor-lite

Reply:
Thanks, I listed both links just in case